ilastik/applets/objectExtraction/objectExtractionSerializer.py

```diff
@@ -95,10 +95,14 @@ def deserialize(self, group):
 class ObjectExtractionSerializer(AppletSerializer):
     def __init__(self, operator, projectFileGroupName):
         slots = [
-            SerialHdf5BlockSlot(operator.LabelOutputHdf5,
-                                operator.LabelInputHdf5,
-                                operator.CleanLabelBlocks,
-                                name="LabelImage"),
+            SerialBlockSlot(operator.LabelImage,
+                            operator.LabelImageCacheInput,
+                            operator.CleanLabelBlocks,
+                            name='LabelImage',
```

Comment:
Same backwards-compatibility concern for this slot, too.  Change the `name` parameter here.